src/app.js

```diff
@@ -48,4 +48,20 @@ $(function () {
             .replace(/Nine/g, "9")
             .replace(/Zero/g, "0"));
     });
+
+    // Media.net ads
+    window._mNHandle = {};
+    window._mNHandle.queue = [];
+    // medianet_versionId = "121199";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3ACommentedCode)